include/convex_bodies/hpolytope.h

```diff
@@ -82,6 +82,29 @@ class HPolytope {
         //_inner_ball = ComputeChebychevBall<NT, Point>(A, b);
     }
 
+   HPolytope& operator = (const HPolytope& other){
```

Comment:
is these needed for this PR?

Reply:
I think it is. We set that this way with Tolis in terms of making the function work..